index.js

```diff
@@ -36,6 +36,7 @@ server.post({ path: '/ProvisionAddress', flags: 'i' }, provisionAddress);
 server.get('/', authCheck);
 server.get({ path: '/GetEndpoints', flags: 'i' }, getEndpoints);
 server.get({ path: '/GetAddressesByDid/:did', flags: 'i' }, getAddressesByDid);
+server.get({ path: '/GetProvisionedAddressByDid/:did', flags: 'i'}, getProvisionedAddressByDid);
```

Comment:
@benjcallaghan, I see you changed the format here. Was there a problem with the regex? I figured there would be eventually. This seems pretty straightforward.